packages/kbn-build/README.md

```diff
@@ -140,11 +140,32 @@ And if needed, you can skip packages in the same way as for bootstrapping, e.g.
 yarn kbn run build --skip-kibana
 ```
 
+## Building packages for production
+
+### "Core" packages (aka packages in the Kibana repo)
+
+As Kibana doesn't have an "install script" in production, we can't run Yarn when
+installing Kibana. Because of that we can't use Yarn to set up symlinks to
+packages (like we do with `link:` dependencies in development). That means we
+either have to prepare the Kibana packages at build time or when Kibana is
+starting up. We decided to prepare Kibana packages for production during the
+build step.
+
+During the build step we build all the Kibana packages required in production,
+and copy them into Kibana's `node_modules` directory, which means it will be
+possible to `import`/`require` these packages in production without symlinks.
+
+### Kibana plugins
```

Comment:
I should probably remove this section for now, and introduce it in https://github.com/elastic/kibana/pull/16481